### PR TITLE
Removed all usage of PokiSDK.SDK

### LINKED ISSUE #1: Breaking usage of internal PokiSDK API
This plugin seems to use internal undocumented PokiSDK APIs. Specifically `PokiSDK.SDK.adTimings`. All of `PokiSDK.SDK` can be changed at any moment without warning. We recently did this and broke all games using this plugin. We have temporarily restored `PokiSDK.SDK.adTimings` but it can be changed again at any moment.

Can you please remove all uses of `PokiSDK.SDK`.

I'm not sure I understand exactly you are using it for. Maybe I can help with some officially supported APIs instead.

I don't know what `this.PostToRuntime("SetCommercialBreakConstraint",{constrained:false});` does or why you sometimes do `PokiSDK.SDK.adTimings.timings.startAdsAfter+10`?

It seems like this code is used to only pause the game when an actual commercial break request is happening: https://github.com/AvixGames/poki-c3-plugin/blob/ad8e10c9e591b6472a15264e25c293f8b692c37a/c3runtime/domSide.js#L102-L103
I understand the use case here and will see if we can officially support something like this. For now I would just always pause the game. If nothing happens it will resume right away anyways.

If you need any help with this please let me know!

### PR DESCRIPTION
**Pull request by skymen**
Fixes https://github.com/AvixGames/poki-c3-plugin/issues/1

This required deprecating OnCommercialBreakPossible in the process

### CODE REVIEW

File: c3runtime/instance.js [hunk 1]
```diff
@@ -94,8 +93,7 @@
 			if (this._automaticSuspend && this._runtime._suspendCount > 0) this._runtime.SetSuspended(false);
 		}
 		SetCommercialBreakConstraint({ constrained }) {
-			this._commercialBreakPossible = !constrained;
-			this.Trigger(self.C3.Plugins.Avix_PokiSDK_ForC3.Cnds.OnCommercialBreakPossible);
+			//this.Trigger(self.C3.Plugins.Avix_PokiSDK_ForC3.Cnds.OnCommercialBreakPossible);
```
Comment (erikdubbelboer) [inline c3runtime/instance.js]: Should this `console.warn` maybe so developers know they should remove calls to this function?

File: c3runtime/domSide.js [hunk 2]
```diff
@@ -99,17 +79,10 @@
         RequestCommercialBreak() {
             if (!this._pokiSDKLoaded) return {result:false,err:false};
             if (this._gameplayActive) this.NotifyGameplayStop();
-            const commercialPossible = this._firstAdTimerDone && PokiSDK.SDK.adTimings.requestPossible();
-            if (commercialPossible) this.PostToRuntime("SuspendRuntime");
+            this.PostToRuntime("SuspendRuntime");
             return PokiSDK.commercialBreak()
                 .then(() => {
                     if (commercialPossible) this.PostToRuntime("ResumeRuntime");
```
Comment (erikdubbelboer) [inline c3runtime/domSide.js]:
```suggestion
                    this.PostToRuntime("ResumeRuntime");
```

**Review by erikdubbelboer — CHANGES_REQUESTED** [review]

File: c3runtime/instance.js (hunk 1 above)
Comment (skymen) [reply c3runtime/instance.js]: No, because this code will never be reached

**Review by ncastez — APPROVED** [review]

**Comment by ncastez** [comment]
Thanks @skymen and @pacoelayudante ! 
Your pull requests was merged into our master

**Pull request merged**